Mathlib/Algebra/Order/Ring/Canonical.lean

```diff
@@ -37,6 +37,9 @@ class CanonicallyOrderedCommSemiring (α : Type*) extends CanonicallyOrderedAddC
   /-- No zero divisors. -/
   protected eq_zero_or_eq_zero_of_mul_eq_zero : ∀ {a b : α}, a * b = 0 → a = 0 ∨ b = 0
 
+-- use `CanonicallyOrderedCommSemiring.toOrderedCommSemiring`
+attribute [instance 0] CanonicallyOrderedCommSemiring.toCommSemiring
```

Comment:
Instead of this change, I'd argue we should change the `extends` line above to use `CanonicallyOrderedCommSemiring`.

Reply:
I think that's ok, and we can use default arguments to make that not be painful.